src/client/pages/Debugger/components/QuoteFormDenmark.tsx

```diff
@@ -13,8 +13,10 @@ export const initialDkHomeValues = {
   danishHomeContents: {
     coInsured: 0,
     livingSpace: 34,
-    street: 'Nørrebrogade 50',
-    zipCode: '1234',
+    street: 'Theodore Roosevelts Vej 1',
+    floor: '2',
+    apartment: 'tv',
```

Comment:
I guess these can still be reset/erased, right?

Reply:
Yes, since they're not required to create a quote the values can be deleted on the `/debugger` page if you want to create a quote with an address that isn't an apartment.

Reply:
And all values in that `debugger` form can of course be edited 😀